Dockerfile

```diff
@@ -8,15 +8,20 @@ RUN apk add --no-cache fontconfig \
  && apk add --no-cache --virtual .build-deps curl git \
  && echo "Fixing PhantomJS to run on alpine" \
  && curl -Ls "https://github.com/tidepool-org/tools/raw/master/alpine_phantomjs_dependencies/dockerized-phantomjs.tar.xz" | tar xJ -C / \
- && mkdir /app/dist && chown node:node -R /app/dist && chmod -R 755 /app/dist \
- && yarn install \
+ && mkdir /app/dist && mkdir /app/node_modules && chown node:node -R /app \
  && apk del .build-deps \
  && rm -rf /usr/share/man /tmp/* /var/tmp/* /root/.npm /root/.node-gyp
 
+USER node
+
 COPY . /app
 
-USER node
+RUN yarn install
+
+VOLUME /app
+VOLUME /app/node_modules
```

Comment:
Does exposing node_modules make sense? Compiled modules wouldn't work across different OSes.

Reply:
You're right, there's no good reason for this.  I've removed it.